website/docs/components/button/examples/one-addon.jsx

```diff
@@ -4,7 +4,7 @@ import CheckM from '@semcore/icon/Check/m';
 
 export default function () {
   return (
-    <Button>
+    <Button ariaLabel="check icon">
```

Comment:
That text is very technical and looks like we are suggesting developers to use techincal and meaningless labels